.github/actions/publish_to_pages_staging/action.yml

```diff
@@ -0,0 +1,12 @@
+name: publish_to_pages_staging
+description: Publish to cloudflare pages (staging)
```

Comment:
Same as here, we might need to set environment variables for `wrangler` to deploy, docs for reference: https://developers.cloudflare.com/workers/wrangler/ci-cd/